src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -82,6 +84,8 @@ public void AssignRowsAndInitializeDisplay()
                     counter.Row = row++;
                 }
             }
+
+            maxRow = row;
```

Comment:
```suggestion
            maxRow = Max(maxRow, row);
```

Reply:
I don't mind, but as far as I can tell providers or counters are never removed. How could `maxRow` ever be bigger than `row`? 🤔

Reply:
You're correct.  This is just futureproofing. I think dynamic counters and TTL on their output is on the table as feature work.

Reply:
Makes sense 👍